modules/openy_features/openy_node/modules/openy_node_social_post/config/install/node.type.social_post.yml

```diff
@@ -0,0 +1,19 @@
+langcode: en
+status: true
+dependencies:
+  enforced:
+    module:
+      - social_feed_fetcher
+  module:
+    - menu_ui
+third_party_settings:
+  menu_ui:
+    available_menus: {  }
+    parent: ''
+name: 'Social Post'
+type: social_post
+description: 'This is a content type for the posts, pulled from Social Networks.'
+help: ''
```

Comment:
Imagine the site will be used by babysitter and write a help for the content type her/him, please

Reply:
Especially because of you already provided a help, just reuse it or add link to github

Reply:
For this comment user has to create advanced_help_block entity. Provide text and set rules of it's visibility. I'm not sure if this help settings can interrupt to create advanced_help_block. I just removed it .